### PR TITLE
move requirements.txt in repo into yaml

### PR DESCRIPTION
**Pull request by hchandola**
1. Move requirements.txt pip packages list into the yml files for github action.
2. Disable automl github action temporarily since the current automl forecasting notebook in master requires manually adding training pipeline run id.

### CODE REVIEW

**Comment by deeptim123** [comment]
can we disable only forecasting and atleast running training for now?

File: .github/workflows/automl-ci.yml [hunk 1]
```diff
@@ -34,7 +34,8 @@ jobs:
           architecture: 'x64'
       - name: Install dependencies
         run: |
-          python -m pip install -r Automated_ML/requirements.txt
+          python -m pip install --upgrade azureml-sdk[automl] \
```
Comment (deeptim123) [inline .github/workflows/automl-ci.yml:37]: actually this is prone to breaking especially during testing. can we take a dependency on AzureMlCli\notebooks\how-to-use-azureml\automated-machine-learning\automl_env.yml somehow.

or atleast install from   https://automlcesdkdataresources.blob.core.windows.net/validated-requirements/AZUREML-SDK-VERSION/validated_win32_requirements.txt [--no-deps]

File: .github/workflows/automl-ci.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-      - 01_Data_Preparation.ipynb
-      - Automated_ML/**
+#on:
+#  pull_request:
+#    branches:
+#      - master
+#    paths:
+#      - 00_Setup_AML_Workspace.ipynb
+#      - 01_Data_Preparation.ipynb
+#      - Automated_ML/**
```
Comment (deeptim123) [inline .github/workflows/automl-ci.yml:10]: i already have a comment elsewhere, but basically keep the training notebook running and make sure atleast that stage gets passed. also if the run is on workspace, we can modify the notebook to not ask for the pipelinerunid.

File: .github/workflows/automl-ci.yml (hunk 1 above)
Comment (deeptim123) [reply .github/workflows/automl-ci.yml:37]: there is a linux version as well.

File: .github/workflows/automl-ci.yml (hunk 2 above)
Comment (hchandola) [reply .github/workflows/automl-ci.yml:10]: I can keep the training notebook in the action but automl team would have to modify their notebook to fix the manually entered pipelinerun id .

File: .github/workflows/automl-ci.yml (hunk 1 above)
Comment (hchandola) [reply .github/workflows/automl-ci.yml:37]: Do you mean the pandas dependency is prone to breaking here? The other azureml packages would be at the same version.

**Comment by hchandola** [comment]
sign off communicated offline by @deeptim123

**Pull request merged**